packages/text-field/src/TextField/TextField.tsx

```diff
@@ -0,0 +1,135 @@
+import { useComposedRefs } from "@telegraph/compose-refs";
+import { Icon } from "@telegraph/icon";
+import { clsx } from "clsx";
+import React from "react";
+
+import { COLOR, SIZE } from "./TextField.constants";
+
+type TextFieldProps = Omit<
+  React.HTMLAttributes<HTMLInputElement>,
+  "value" | "onChange"
+> & {
+  size?: "1" | "2" | "3";
+  variant?: "outline" | "ghost";
+  disabled?: boolean;
+  error?: boolean | string;
```

Comment:
Does this component need to care about an error message? i.e. string value?

Reply:
Yes, the border of the TextField turns red when there is an error.

Reply:
`error` is being passed from `Field` which is a `string` or `boolean` value. `string` when there is an error message `boolean` when there is no message but an error state is triggered. Thus, it would be incorrectly typed in `TextField` if we constrained this to just a boolean when it can be a string.